scripts/postinst

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh

set -e
```

See [Debian Policy Manual 6.1](https://www.debian.org/doc/debian-policy/ch-maintainerscripts.html#introduction-to-package-maintainer-scripts) regarding `set -e`.

I cannot find an entry in the manual regarding selection of shells for control scripts; however, the packages I surveyed on my machine only calls `sh`. I believe there is a guarantee that sh in installed and behaves correctly. It seems likely bash will always exist, but I don't think it's guaranteed.

Reply:
Addressed: b6007e022d and [61b0efe](https://github.com/AntelopeIO/leap/pull/818/commits/61b0efe536f629077bcde91caf93b062c2749eb7)